src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -1,10 +1,13 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Generic;
 using System.Globalization;
 using System.IO;
 using System.Linq;
+using System.Threading.Tasks;
```

Comment:
I don't think that System.Threading.Tasks is needed. At least, it's not obvious to me from the other changes in the file why it's needed. If it's no longer needed, it would be nice to remove unneeded usings.

Reply:
Removed